samples/bluetooth/peripheral_uart/prj.conf

```diff
@@ -39,3 +44,12 @@ CONFIG_DK_LIBRARY=y
 
 # This example requires more workqueue stack
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+# Config logger
+CONFIG_LOG=y
+CONFIG_LOG_MINIMAL=n
```

Comment:
This is the default

Reply:
Default value is `y`. This have to be disabled here

Reply:
It is turned on by `CONFIG_NCS_SAMPLES_DEFAULTS`. I am unsure whether we should have that config turned on and override part of the configs, or not turn it on at all, and set what we need manually. I am more inclined toward the latter, it seems clearer to me.

Reply:
Ok in this sample I removed `CONFIG_NCS_SAMPLES_DEFAULTS` so legger configuration should be clearer